src/condor_utils/file_transfer.cpp

```diff
@@ -6576,44 +6574,44 @@ FileTransfer::InvokeMultipleFileTransferPlugin( CondorError &e,
 		plugin_args.AppendArg( "-upload" );
 	}
 
-	// Invoke the plugin
-	dprintf( D_ALWAYS, "FILETRANSFER: invoking: %s \n", plugin_path.c_str() );
-	dprintf( D_FULLDEBUG, "FILETRANSFER: INPUT FILE: %s\n", transfer_files_string.c_str() );
-	FILE* plugin_pipe = my_popen( plugin_args, "r", FALSE, &plugin_env, drop_privs );
-	if( !plugin_pipe ) {
-		dprintf ( D_ALWAYS, "FILETRANSFER: failed to invoke multifile transfer "
-			"plugin %s, aborting\n", plugin_path.c_str() );
-		return TransferPluginResult::Error;
-	}
+	bool want_stderr = param_boolean("REDIRECT_FILETRANSFER_PLUGIN_STDERR_TO_STDOUT", true);
```

Comment:
Strictly speaking, we should also be doing this for single-file plug-ins.

Reply:
stdout of single file plugins is defined to be the status ad.